### DISCUSSION TITLE
Log data with Traces and Spans

### DISCUSSION THREAD

**Discussion by alokmajumder** [open-telemetry/opentelemetry-java#2883] [Q&A]
I am able to export span and traces, but not the logs to zipkin or jaeger. I want to see the logs for each span along with spans. Any example ?
I am  using otel exporter.

**Comment by jkwatson** [comment]
Hi @alokmajumder . OpenTelemetry logs is still a work-in-progress and we don't have any official support at this point. There is a very-early prototype module `opentelemetry-sdk-extension-logging` if you want to try it out.
